typescript/packages/well-log-viewer/src/SyncLogViewer.stories.tsx

```diff
@@ -158,40 +158,41 @@ function fillInfo(controller) {
 
 const Template = (args) => {
     const infoRef = React.useRef();
-    const setInfo = function (info) {
+    const setInfo = function (info: string) {
         if (infoRef.current) infoRef.current.innerHTML = info;
     };
 
-    const [controller, setController] = React.useState(null); // the first WellLog
     const [controllers, setControllers] = React.useState<WellLogController[]>(
         []
     ); // all WellLogs
 
     const onCreateController = React.useCallback(
-        (iWellLog, controller) => {
-            if (iWellLog === 0) setController(controller);
-
+        (iWellLog: number, controller: WellLogController) => {
             setControllers((prev) => [...prev, controller]);
         },
-        [controller]
+        [controllers]
+    );
+    const onDeleteController = React.useCallback(
```

Comment:
What does this fix. Probably not React.Strict.
Can you add a comment in the PR description ?

Reply:
add 4-th item into PR description about new onDeleteController callback